cargo-concordium/src/main.rs

```diff
@@ -169,8 +200,9 @@ enum RunCommand {
         #[structopt(
             name = "state-json",
             long = "state-json",
-            help = "File with existing state of the contract in JSON, requires a schema is \
-                    present either embedded or using --schema."
+            help = "File with existing state of the contract in JSON,
+requires a schema is present either embedded or using
+--schema."
```

Comment:
I did not change the formatter. This was probably caused the time you commented out everything, the formatter then formatted the comment, and then did not reformat since it was a mutline string already.

Reply:
Ah, I see. So I'm the troublemaker 😅